packages/itwinui-css/src/badge/badge.scss

```diff
@@ -1,27 +1,23 @@
 // Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 // See LICENSE.md in the project root for license terms and full copyright notice.
-@import '../style/index';
 @import '../utils/index';
 
-$iui-badge-background-default: $iui-color-background-5--light;
-$iui-badge-background-positive: $iui-color-foreground-positive--dark;
-$iui-badge-background-negative: $iui-color-foreground-negative--dark;
-$iui-badge-background-warning: $iui-color-foreground-warning--dark;
-$iui-badge-background-primary: $iui-color-foreground-primary--dark;
-$iui-badge-horizontal-padding: $iui-xs * 1.5;
-
-$iui-badge-text-color: rgba($iui-color-foreground-body--light, $iui-opacity-3--light);
+$iui-badge-background-positive: hsl(96, 45%, 78%); // foreground-positive--dark
```

Comment:
we don't have `--dark` or `--light` versions of variables anymore so i hardcoded it, because currently our badges are supposed to look the same across themes.

in the future, i think we want the `soft-background` colors to be themeable so we might be able to use those